lndr.module

```diff
@@ -1,79 +1,94 @@
-      $output .= '<dd>' . t('You can find the API token for Lndr under your <a href="@lndr-user">user profile in Lndr</a>.', ['@lndr-user' => 'https://www.lndr.co/users/edit']) . '</dd>';
+      $output .= '<dd>' . t('You can find the configuration page for Lndr under Administration > Configuration > Content authoring or <a href="@lndr">Lndr configuration page</a>.', array('@lndr' => base_path() . 'admin/config/content/lndr')) . '</dd>';
+      $output .= '<dd>' . t('You can find the API token for Lndr under your <a href="@lndr-user">user profile in Lndr</a>.', array('@lndr-user' => 'https://www.lndr.co/users/edit')) . '</dd>';
       $output .= '<dt>' . t('Using the module') . '</dt>';
       $output .= '<dd>' . t('Generally, you do not need to actively use this module once it has been successfully configured. There might be some exceptions such as changing the API token.') . '</dd>';
       $output .= '<dd>' . t('If your landing page created for Drupal does not show up, you can always run the Drupal cron manually to trigger the page sync.') . '</dd>';
-      $output .= '<dd>' . t('Your landing page created for Drupal can be found under the <a href="@url-alias">URL alias</a> section', ['@url-alias' => base_path() . 'admin/config/search/path']) . '</dd>';
+      $output .= '<dd>' . t('Your landing page created for Drupal can be found under the <a href="@url-alias">URL alias</a> section', array('@url-alias' => base_path() . 'admin/config/search/path')) . '</dd>';
       $output .= '<dd>' . t('Lndr pages in Drupal will appear in the URL alias as /lndr/[id]') . '</dd>';
       $output .= '</dl>';
-      $output .= '<p>' . t('For more information, contact <a href="@lndr-support">support@lndr.co</a>', ['@lndr-support' => 'mailto:support@lndr.co']) . '</p>';
+      $output .= '<p>' . t('For more information, contact <a href="@lndr-support">support@lndr.co</a>', array('@lndr-support' => 'mailto:support@lndr.co')) . '</p>';
       return $output;
   }
 }
 
-
 /**
  * Implements hook_init
- * Enable cross-domain request for the API path
+ * Enable cross-domain request for the API path.
  */
 function lndr_init() {
+drush_print_r(__FUNCTION__);
 
-  // Only add the CORS header to our service path for security
+
+  // Only add the CORS header to our service path for security.
   $current_path = drupal_strtolower(drupal_get_path_alias($_GET['q']));
   $page_match = drupal_match_path($current_path, 'service/lndr/*');
   if ($current_path != $_GET['q']) {
     $page_match = $page_match || drupal_match_path($_GET['q'], 'service/lndr/*');
   }
   if ($page_match) {
-    // Let's add our CORS headers
+    // Let's add our CORS headers.
     $headers = array(
-      'Access-Control-Allow-Origin' => '*',
-      'Access-Control-Allow-Credentials' => true,
+      'Access-Control-Allow-Origin' => 'https://lndr.co',
```

Comment:
We have to leave it * for now, or *.lndr.co or something because of staging.lndr.co, dev.lndr.co. Also, module had issues when we restricted origin.

Reply:
done